pkgs/tools/virtualization/cloud-init/default.nix

```diff
@@ -19,15 +20,17 @@ in pythonPackages.buildPythonApplication rec {
       --replace /etc $out/etc \
       --replace /lib/systemd $out/lib/systemd \
       --replace 'self.init_system = ""' 'self.init_system = "systemd"'
+
+    patchPhase
```

Comment:
Why wouldn't it? It does work AFAICT (my understanding is that this will simply call the original `prePatch` script, won't it?). But `prePatch` is probably cleaner.